docker/pravega/entrypoint.sh

```diff
@@ -26,12 +26,30 @@ configure_controller() {
     echo "JAVA_OPTS=${JAVA_OPTS}"
 }
 
-configure_segmentstore() {
-    add_system_property "pravegaservice.zkURL" "${ZK_URL}"
-    add_system_property "autoScale.controllerUri" "${CONTROLLER_URL}"
+configure_nfs_s3_hdfs() {
+    add_system_property "tier2.enableNfs" "${ENABLE_NFS}"
+
+    echo "Trying to check whether NFS mounting is required"
+    if [ "${ENABLE_NFS}" = "true" ] && [ "${MOUNT_IN_CONTAINER}"  = "true" ]; then
+    NFS_MOUNT=${NFS_MOUNT:-"/nfs/"}
+        echo "Mounting the NFS share"
+        mkdir -p ${NFS_MOUNT}
+        while [ true ]
+        do
+            mount -t nfs ${NFS_SERVER} ${NFS_MOUNT} -o nolock && break
```

Comment:
Similar to NFS_MOUNT, I think it would be possible for `NFS_SERVER` to be unset and this would never succeed. It might be better to add this to the condition or handle that error explicitly.